translations/get5.phrases.txt

```diff
@@ -140,6 +140,11 @@
         "#format"       "{1:s}"
         "en"            "Sorry, team {1} has no more tech pause time. Please use tactical pauses."
     }
+    "TechPausePausesRemaining"
+    {
+        "#format"       "{1:s},{2:d}"
+        "en"            "Team {1} has {2} technical pauses remaining."
```

Comment:
I would suggest `Team {1} has {2} technical pause(s) remaining.` so it doesn't look silly if they have 1 pause remaining.